### ISSUE TITLE
Url::getAbsoluteUrl() has unexpected output - missing // in scheme

### ISSUE THREAD

**Issue by JanMikes** [nette/http#171]
Hi, i am not sure, this just might be undocumented BC break (i was not able to find it in changelog) or we are doing something wrong.

Version: >=3.0.0

### Bug Description
Following code worked in 2.4.11, but in 3.0.2 has output `glamitrends:products` which obviously is not valid url address.

```php
$url = new Nette\Http\Url();
$url->setScheme('glamitrends');
$url->setPath('products');

Assert($url->getAbsoluteUrl() === 'glamitrends://products');
```

### Expected Behavior
`$url->getAbsoluteUrl()` output to be `glamitrends://products'`

**Comment by JanMikes** [comment]
This workaround (code change) seems to be working for us:

```php
$url = new Nette\Http\Url('glamitrends://products');
```

**Comment by JanTvrdik** [comment]
In `glamitrends://products`, the `products` part is a hostname, not a path.

**Comment by dg** [comment]
It was broken in 2.4 (fix 8ffa1cd5f1db2d1e4e1d93464a47753175c27a1c)

**Comment by JanMikes** [comment]
Ok, thank you for explaining. I understand it that way, we were exploiting a bug, thinking it is a feature and since it was fixed it no longer works for us 😄
